### ISSUE TITLE
up and down arrow keys in Buffers command

### ISSUE THREAD

**Issue by vitmy0000** [junegunn/fzf.vim#513]
- Category
    - [X] Question
    - [ ] Bug
    - [ ] Suggestion
- OS
    - [ ] Linux
    - [X] macOS
    - [ ] Windows
    - [ ] Etc.
- Vim
    - [X] Vim
    - [ ] Neovim

Looks like up and down arrow keys are not working in command `:Buffers`.
Is there any way to solve this?

**Comment by junegunn** [comment]
Works for me, the first entry in `:Buffers` is the current buffer, so we are disallowing selecting it by using `--header-lines 1` option of fzf.

**Comment by nkcfan** [comment]
Same to me. I have only 2 buffers.
@junegunn could you please reopen this issue?

**Comment by nkcfan** [comment]
Not sure I met the exactly the same issue. I opened two files, and would like to use key mapping of `:Buffers` to switch them. BTW, I am using vim8 with fzf popup option.

I notice that I cannot switch to the other buffer after several switching.

I tuned the code to use `autocmd BufWinEnter,WinEnter * let g:fzf#vim#buffers[bufnr('')] = localtime()`, most of the time it works, however, it fails if I switch too quickly.

I did not notice the problem happens with >= 3 buffers.
